lib/components/first-time-point/style.less

```diff
@@ -5,6 +5,8 @@
 
   @{prefix}arrow {
     position: absolute;
+    left: -45px;
```

Comment:
Now that `dqpl-arrow` defaults to these top and left values, you can remove [`left` and `top` from here](https://github.com/dequelabs/pattern-library/pull/95/files#diff-9358e49477fe9294376283d27fb0416aR13)